### PR TITLE
Loading button

### PR DESCRIPTION
**Pull request by Caleb335**
@shahednasser I've worked on the loading button class, I also added some other loading variation.

Take a look at it in the GIF below and lemme know what you think. I await your review, Thanks.

Issue: #436 

ezgif com-gif-maker (4)

### CODE REVIEW

File: src/components/animated/_loading-btn.less [hunk 1]
```diff
@@ -0,0 +1,188 @@
+.loading-btn {
+//   &.yellow-btn {
```
Comment (shahednasser) [inline src/components/animated/_loading-btn.less:2]: Please remove the comment

**Review by shahednasser — CHANGES_REQUESTED** [review]
Love what you've done! I just have one request below

File: src/components/animated/_loading-btn.less (hunk 1 above)
Comment (Caleb335) [reply src/components/animated/_loading-btn.less]: All done.

**Comment by shahednasser** [comment]
Great work thanks!

**Comment by Caleb335** [comment]
> Great work thanks!

You're welcome @shahednasser.

**Pull request merged**